### DISCUSSION TITLE
Upgrade pg_cron to version 1.6

### DISCUSSION THREAD

**Discussion by devidevio** [supabase/supabase#18543] [Feature Requests]
Is it possible to upgrade the database extension `pg_cron` to version 1.6 which was released a few days ago?
* https://www.citusdata.com/blog/2023/10/26/making-postgres-tick-new-features-in-pg-cron/

The latest available version in Supabase is v1.4-1:
| name    | version |
| ------- | ------- |
| pg_cron | 1.0     |
| pg_cron | 1.1     |
| pg_cron | 1.2     |
| pg_cron | 1.3     |
| pg_cron | 1.4     |
| pg_cron | 1.4-1   |

`SELECT name, version FROM pg_available_extension_versions WHERE name = 'pg_cron'`

Thanks in advance.

**Comment by GaryAustin1** [comment]
FYI  They tried and had issues...
https://github.com/supabase/postgres/pull/652
